addons/payment/controllers/portal.py

```diff
@@ -355,12 +354,12 @@ def _update_landing_route(tx_sudo, access_token):
                                 f'?tx_id={tx_sudo.id}&access_token={access_token}'
 
     @http.route('/payment/confirmation', type='http', methods=['GET'], auth='public', website=True)
-    def payment_confirm(self, tx_id, access_token, **kwargs):
+    def payment_confirm(self, tx_id, access_token, **_kwargs):
```

Comment:
Why? Your linter shouldn't tell you anything about this one

Reply:
Doesn't yours show a warning?

pylint 2.4.4-2 does show a warning for this line -> `addons/payment/controllers/portal.py:357:0: W0613: Unused argument 'kwargs' (unused-argument)`
This param is indeed unused, but we keep it for eventual community overrides.

Reply:
No, I am not sure why, unused method params are just grayed out while inside method unused var indeed raise a linter error.
There is not a single occurence of `**_` in the odoo codebase, (except the 2 in payment), so I am not sure if you configured something or if this is right, but clearly something I never saw before (it's not rare that kwargs is not used, or even regular method param in "placeholder" method meant to be override).
Thought I would mention it.
![image](https://user-images.githubusercontent.com/30048408/190607114-d8a3edf4-6353-4a13-b47a-4d4684687e4b.png)

Reply:
I didn't configure anything in particular regarding pylint so I guess it's normal.

My team and I did configure our IDEs to detect method overrides and consider "locally" unused (kw)args as *used*, though. That's maybe why there are so few occurrences of `**_` in the codebase. I personally don't consider it a bad practice as it immediately tells that the kwarg is indeed not used anywhere in standard Odoo, so I'd rather keep it like this.^^

Reply:
> so I'd rather keep it like this.^^

Well since I don't have any valid argument to not do it, but just some personal opinion that's fair ^^:
- We don't do that in the code base
- I would not do it myself
- As args and kwargs and kind of special arguments, it makes sense to not consider those as really unused, a bit like self, even if this one is a special case in pylint to not consider it unused ever

Reply:
> @JKE-be It is not wrong but make no plusvalue, except to increase the risk of error, and the need to double check each time the logger name... (and in stable conflict if you have a local diff using it)

Well, here is your valid argument :p I'll remove that part of the diff. Interesting debate, though :)